package.json

```diff
@@ -8,7 +8,7 @@
     "url": "git+https://github.com/gridstack/gridstack.js.git"
   },
   "scripts": {
-    "build": "grunt && doctoc ./README.md && doctoc ./doc/README.md && doctoc ./doc/CHANGES.md",
+    "build": "mkdir dist && grunt && doctoc ./README.md && doctoc ./doc/README.md && doctoc ./doc/CHANGES.md",
```

Comment:
hey guys I believe this is blocking the pipeline as mkdir dist will give an error if folder is already there (and build seems to create it). I removed it a subsequent review (which hasn't been been merged yet (see #982)